transformers/tokenization_utils.py

```diff
@@ -282,6 +283,9 @@ def from_pretrained(cls, *inputs, **kwargs):
             # Download vocabulary from S3 and cache.
             tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
 
+            # Download vocabulary from S3 (user-uploaded) and cache.
+            tokenizer = BertTokenizer.from_pretrained('dbmz/bert-base-german-cased')
```

Comment:
@julien-c Sorry for the late reply on that PR! The prefix should be `dbmdz` here :D

Reply:
Ooopsie :)

I'll fix as I write more doc everywhere this feature is supported.